### ISSUE TITLE
Integrate methods for bootstrapping

### ISSUE THREAD

**Issue by vasanchez16** [vasanchez16/smokeppe_constraints#7]
Implement methods for bootstrap estimation for both the pivotal and non-pivotal cases. First work will be done to implement the scripts for this. Then there will be work done to parallelize some of the methods.

**Comment by vasanchez16** [comment]
@JamesCarzon , 

Just wanted to let you know I took the nu out of the bootstrap calculations like we discussed in meeting. I am also going to be removing the pivotal and non pivotal options for the bootstrap since we only will be using one method here

**Comment by JamesCarzon** [comment]
I see what you mean. You are right -- it should stay for the non-bootstrapped example since the simulated null distribution is drawing T and not sqrt((nu-2)/nu)*T. However, in my opinion these should be rescaled in both `implausibility/implausibility.py` and `confidence/utils.py` for consistency.

**Comment by vasanchez16** [comment]
I'm not sure I understand. For the student-t bootstrap, the nu factor is not included in the bootstrap and not included in the implausibility calculations. For the regular student-t assumption, the nu factor is accounted for in the implausibility and not needed in the threshold because we are pulling from a regular student t distribution. 

Do you mind pointing out the lines of code that are problematic?

**Comment by JamesCarzon** [comment]
I agree with you. The calculation looks good to me. I'm just suggesting not rescaling the implausibility and instead rescaling the simulated noise for the reason I think it's more consistent with how the bootstrap alternative is implemented. But this is an aesthetic judgment. We should leave it as is for now and perhaps worry later if you want to publish the code.

**Comment by vasanchez16** [comment]
ok sounds good we can look at this again later on
